### PR TITLE
Add bug and feature templates

### PR DESCRIPTION
**Pull request by vicb**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+title: "[BUG]"
+labels: triage
+assignees: ""
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior (linking a repo with a reproduction will help for non trivial issues):
```
Comment (petebacondarwin) [inline .github/ISSUE_TEMPLATE/bug_report.md:13]:
```suggestion
Minimal steps to reproduce the behavior (linking a repo with a reproduction will help for non trivial issues):
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Environment (please complete the following information):**
+
+- @opennextjs/cloudflare version:
+- Node version:
```
Comment (petebacondarwin) [inline .github/ISSUE_TEMPLATE/bug_report.md:21]: Possibly Wrangler version?

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: "[FEATURE]"
+labels: enhancement, triage
+assignees: ""
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is.
```
Comment (petebacondarwin) [inline .github/ISSUE_TEMPLATE/feature_request.md:10]: Do we want to encourage the use of GH discussions for more complex or controversial feature requests? Or perhaps we can just keep it like this and move the issue to a discussion if it feels like it is going to lead to wide ranging discussion?

**Review by petebacondarwin — APPROVED** [review]
One small suggestion and a couple of comments.

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 3 above)
Comment (vicb) [reply .github/ISSUE_TEMPLATE/feature_request.md:10]: +1 to 

> perhaps we can just keep it like this and move the issue to a discussion if it feels like it is going to lead to wide ranging discussion?

(and we can improve the templates when the need arises)

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Minimal steps to reproduce the behavior (linking a repo with a reproduction will help for non trivial issues):
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Environment (please complete the following information):**
```
Comment (dario-piotrowicz) [inline .github/ISSUE_TEMPLATE/bug_report.md:18]: A good idea could be to also ask them to run `next info`? (like in the [Next.js repo](https://github.com/vercel/next.js/blob/224447cd65d23f2dec2119b0c9ef10b79532dc6e/.github/ISSUE_TEMPLATE/1.bug_report.yml?plain=1#L51-L75))

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG]"
+labels: triage
```
Comment (dario-piotrowicz) [inline .github/ISSUE_TEMPLATE/bug_report.md:5]:
```suggestion
labels: bug, triage
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+assignees: ""
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Minimal steps to reproduce the behavior (linking a repo with a reproduction will help for non trivial issues):
```
Comment (dario-piotrowicz) [inline .github/ISSUE_TEMPLATE/bug_report.md:13]:
```suggestion
Minimal steps to reproduce the behavior (linking a repo with a reproduction will help for non trivial issues).
```

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 4 above)
Comment (dario-piotrowicz) [inline .github/ISSUE_TEMPLATE/bug_report.md:18]:
```suggestion
**Environment (please complete the following information)**
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+
+**Describe the solution you'd like**
+A clear and concise description of what you want to happen.
+
+**Describe alternatives you've considered**
+A clear and concise description of any alternative solutions or features you've considered.
+
+**Additional context**
+Add any other context about the feature here.
+What @opennextjs/cloudflare does this apply to?
```
Comment (dario-piotrowicz) [inline .github/ISSUE_TEMPLATE/feature_request.md:20]: what do you mean with this question? what version of the package?

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 4 above)
Comment (vicb) [reply .github/ISSUE_TEMPLATE/bug_report.md:18]: Very valid point.
At some point we should switch to the form version of the template.
I'll create an issue for that.

**Review by dario-piotrowicz — APPROVED** [review]
looks good to me

I think using headings instead of bolded text would be nicer but it's also pretty minor

**Pull request merged**